datumaro/components/extractor.py

```diff
@@ -519,8 +558,36 @@ class DatasetItem:
     annotations = attrib(factory=list, validator=default_if_none(list))
     subset = attrib(converter=lambda v: v or DEFAULT_SUBSET_NAME, default=None)
     path = attrib(factory=list, validator=default_if_none(list))
-
     image = attrib(type=Image, default=None)
+    pcd = attrib(type=bytes, default=None)
```

Comment:
How about a new annotation type for this?

Reply:
@zhiltsov-max : We have added a new annotation Cuboid, we can rename that to pcd, will that solve the purpose?
Kindly confirm if our understanding is correct on the above.

Reply:
I can't see a reason to have points as a part of both cuboids and polygons, because they are different, but it seems like a good candidate for a new annotation type.

Reply:
Hi @zhiltsov-max : We did not understand the above mentioned point, so does it mean what is implemented currently need not be changed or anything should be added, kindly confirm.

Reply:
I expect either an explanation, why pcd logic must be in polygons, or creation of a new subclass `PcdPolygon`.

Reply:
Hi @zhiltsov-max : The pcd logic is not inside polygons here and it has been added as a dataset in Dataset class.

Reply:
Ah, you're right. GitHub layout confused me.